src/jit/flowgraph.cpp

```diff
@@ -623,7 +623,7 @@ GenTreeStmt*  Compiler::fgInsertStmtNearEnd(BasicBlock* block, GenTreePtr node)
 {
     GenTreeStmt* stmt;
     
-    // This routine is not aware of embedded stmts and can only be used when in tree order.
+    // This routine can only be used when in tree order.
     assert(fgOrder == FGOrderTree);
```

Comment:
This assert should probably be based on `compRationalIRForm` rather than `fgOrder`, but I think that's out of scope for this PR.